client/src/css/Jessica.css

```diff
@@ -0,0 +1,55 @@
+h2 {
+  font-size: 60;
+  text-align: center;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.name {
+  text-align: center;
+  color: #fdf1e8;
+  font-size: 70px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  position: absolute;
+  top: 150px;
+  left: 0;
```

Comment:
in general it's best to avoid absolute positioning. there's a way we can use relative positioning for your name, let's discuss this tonight